api/app/signals/apps/xperimental/tests/test_filters.py

```diff
@@ -0,0 +1,1378 @@
+# SPDX-License-Identifier: MPL-2.0
+# Copyright (C) 2021 Gemeente Amsterdam
```

Comment:
Super tiny nitpick, not a blocker; I think copying a file does not reset the copyright on it.

Reply:
File is not copied 😉 however the tests are because we want the new endpoint to give the same results